app/javascript/components/party/Header.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Layout, Row, Col, Typography, Divider, Avatar, Button } from "antd";
+import SocialSharing from "../common/SocialSharing";
+
+const { Title, Paragraph } = Typography;
+
+const Header = ({ party }) => (
+  <Layout>
+    <Layout.Header className="party-header">
+      <Row>
+        <Col span={24}>
+          <Title>{party.name}</Title>
+          <Divider />
+          <Paragraph strong>{party.acronym}</Paragraph>
+        </Col>
+      </Row>
+      <Row type="flex" justify="center">
+        <Col>
+          <Avatar size={250} icon="user" />
+        </Col>
+      </Row>
+      <Row type="flex" justify="center">
+        <Col>
+          <Button className="button--grey">Ver Programa</Button>
```

Comment:
We can hide this for now. We will only introduce the program later on.

Reply:
Sure